de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
Oh! I was thinking deleting a list will not impact permissions. Right now, If I remove a person from my collaborators, it will not un-share items with that collaborators. Is there a specific reason we are changing the existing functionality ?